_data/build-research.py

```diff
@@ -44,22 +43,14 @@
   # paper publisher
   new_paper["publisher"] = old_paper.get("publisher", "")
   # paper date
-  date = old_paper.get("issued").get("date-parts")[0]
-  if len(date) > 0:
-    new_paper["year"] = date[0]
-  if len(date) > 1:
-    new_paper["month"] = date[1]
-    new_paper["month_name"] = calendar.month_abbr[new_paper["month"]]
-  if len(date) > 2:
-    new_paper["day"] = date[2]
+  date_parts = old_paper.get("issued").get("date-parts")[0]
```

Comment:
```suggestion
  date_parts = old_paper.get("issued").get("date-parts")
```
I think a stray bracket there

Reply:
😭